src/components/Menu/MenuItem/MenuItem.module.scss

```diff
@@ -37,6 +37,31 @@
   color: var(--icon-color);
 }
 
+.subMenuIconWrapper .divider {
+  height: 19px;
+}
+
+.subMenuIconWrapper .subMenuSplitButtonIcon {
+  width: 34px;
+  height: 30px;
+  --primary-background-hover-color: rgba(103, 104, 121, 0.1);
+
+  &:hover,
+  &:active {
+    background-color: var(--primary-background-hover-color);
+  }
+}
```

Comment:
When you navigate with the keyboard `IconButton` get `active` state, which looks a bit weird
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/9ca0229e-0995-4374-92f3-b1e266076566)

Color is coming from `primary-selected-color` - I guess it also should be overriden with this value `rgba(103, 104, 121, 0.1)` but maybe worth be checking with designers

Reply:
Meirav wants that the icon button will stay active when the sub menu opens. Today Tal and I found out that active color doesn't have grey color (same color as hovered). He said that he will insert a fix

Reply:
So he'll fix it? Why? Don't you just need to override it for this specific case, no?